### ISSUE TITLE
Unable to queue music

### ISSUE THREAD

**Issue by jumban** [vdesabou/alfred-spotify-mini-player#593]
In the past, I was able to queue songs using Alt+CMD+Enter, including when the music was playing on my computer. 

After updating to v12.4, I now get this error instead when I try to add a song to queue. 

<img width="380" alt="image" src="https://user-images.githubusercontent.com/32335357/208018015-db2aeb9e-0935-4cdd-8eab-c3d24aa163d0.png">

**Comment by vdesabou** [comment]
You need to use Spotify Connect to use this. You probably need to select it again

**Comment by jumban** [comment]
Thanks for the prompt reply. It is already selected under Spotify Connect Devices. 

<img width="713" alt="image" src="https://user-images.githubusercontent.com/32335357/208021408-77e0a757-8e1f-4f43-aa90-e20fd5bd7fa8.png">

**Comment by jumban** [comment]
Just as a test, I have switched to playing on my KEF speakers (i.e. not going through my computer anymore so it's definitely using Spotify Connect). 

<img width="278" alt="image" src="https://user-images.githubusercontent.com/32335357/208021625-c4c82cc5-b755-483e-9cde-c734139f15bf.png">

<img width="707" alt="image" src="https://user-images.githubusercontent.com/32335357/208021597-f7ec3ea9-6ee6-4748-bb10-fde06c5e9c10.png">

It's still showing the same error when I try to queue songs.

**Comment by jumban** [comment]
Hi, just as an update, I did more poking around in the settings and I found that the issue was fixed after I set this option again. 

<img width="710" alt="image" src="https://user-images.githubusercontent.com/32335357/208022167-ed615cc1-a38b-4c63-aea4-8eaba6bb7304.png">

Somehow after this update the setting was reset to "Spotify Desktop" instead of "Spotify Connect". Changing it back to "Spotify Connect" fixed the issue. I can now queue songs again while listening on desktop. 

Thank you!
